NGitLab/Impl/MilestoneClient.cs

```diff
@@ -6,13 +6,31 @@ namespace NGitLab.Impl
 {
     public class MilestoneClient : IMilestoneClient
     {
+        internal enum Scope
+        {
+            Project,
+            Milestone,
+        }
+
         private readonly API _api;
         private readonly string _milestonePath;
 
-        public MilestoneClient(API api, int projectId)
```

Comment:
I would leave this ctor in for now, but tag it as obsolete. Basically, people should go through the GitLab client to instantiate a MilestoneClient.